Bottom: 0991fb3a7659b354aae0470e822cffa94632443d
Top:    a4eac701169d2d22556319be68ef72952ad029d3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:29 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c8a0d8e..f79622e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1234,9 +1234,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
